### PR TITLE
[Pooling] refine pooling ceil_mode

### PR DESCRIPTION
**Pull request by xuhongyao**

### CODE REVIEW

**Review by youbeny — APPROVED** [review]

**Review by ahuizxc — APPROVED** [review]

File: lib/transforms/caffeextension_legalizer.cc [hunk 1]
```diff
@@ -178,7 +178,7 @@ static std::vector<Def> ConvertPool(const CAFFEExtensionInst* ext,
   int pad_h = FindAttributeValue(ext, "pad_h", pad);
   int pad_w = FindAttributeValue(ext, "pad_w", pad);
 
-  int round_mode = FindAttributeValue(ext, "round_mode", -1);
+  int round_mode = FindAttributeValue(ext, "round_mode", 0);
```
Comment (weimingzha0) [inline lib/transforms/caffeextension_legalizer.cc:181]: use "1" as default value? So in line 206, we can setRoundMode(round_mode)

File: lib/transforms/caffeextension_legalizer.cc [hunk 2]
```diff
@@ -194,7 +194,7 @@ static std::vector<Def> ConvertPool(const CAFFEExtensionInst* ext,
     pad_w = 0;
   }
 
-  auto set_pooling_attributes = [&](auto inst, int round_mode) {
+  auto set_pooling_attributes = [&](auto inst) {
```
Comment (weimingzha0) [inline lib/transforms/caffeextension_legalizer.cc:197]: change the calling site?

File: lib/transforms/caffeextension_legalizer.cc (hunk 1 above)
Comment (xuhongyao) [reply lib/transforms/caffeextension_legalizer.cc:181]: the default  value of caffe is 0, which means ceil mode . so if we really get "0", "setRoundMode(round_mode)" is wrong

File: lib/transforms/caffeextension_legalizer.cc (hunk 2 above)
Comment (xuhongyao) [reply lib/transforms/caffeextension_legalizer.cc:197]: yes

File: lib/transforms/caffeextension_legalizer.cc (hunk 1 above)
Comment (weimingzha0) [reply lib/transforms/caffeextension_legalizer.cc:181]: ok

**Pull request merged**